website/client/components/notifications.vue

```diff
@@ -204,6 +203,9 @@ export default {
     invitedToQuest () {
       return this.user.party.quest.RSVPNeeded && !this.user.party.quest.completed;
     },
+    userExpAndLvl () {
+      return `${this.user.stats.exp}|${this.user.stats.lvl}`;
```

Comment:
could you check if its possible to use an array of those values instead of an concated string?